seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/textFormatIT/kafka_source_text_to_console.conf

```diff
@@ -35,23 +35,84 @@ source {
     kafka.auto.offset.reset = "earliest"
     format_error_handle_way = fail
     schema = {
-      fields {
-        id = bigint
-        c_map = "map<string, smallint>"
-        c_array = "array<tinyint>"
-        c_string = string
-        c_boolean = boolean
-        c_tinyint = tinyint
-        c_smallint = smallint
-        c_int = int
-        c_bigint = bigint
-        c_float = float
-        c_double = double
-        c_decimal = "decimal(2, 1)"
-        c_bytes = bytes
-        c_date = date
-        c_timestamp = timestamp
+      columns = [
```

Comment:
Could you create new conf not replace old one? I think we should make sure both two config way worked fine for now.

Reply:
Agree with you